src/widgets/partnered/ui/partnered.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+
+import { Partnered } from './partnered';
+import aws from '@/shared/assets/icons/aws-gray.svg';
+import github from '@/shared/assets/svg/github.svg';
+import jetbrains from '@/shared/assets/svg/jetbrains.svg';
+
+describe('Partnered component', () => {
+  it('should render the component with correct title', () => {
+    render(<Partnered />);
+
+    expect(screen.getByText('Partnered with')).toBeInTheDocument();
+
+    expect(screen.getByTestId('widget-title')).toBeInTheDocument();
```

Comment:
Can you look tests for AboutCourse, for example. How we checkin the content. 
`const title = await screen.findByTestId('widget-title');
expect(title).toBeVisible();
expect(title.textContent).toBe('About the course');
`

Reply:
I rewrote the test for checking the correct content to be similar to AboutCourse one